packages/terra-clinical-result/src/flowsheet-result-cell/FlowsheetResultCell.jsx

```diff
@@ -215,7 +225,7 @@ const unpackResultAttributes = (resultDataItem) => {
     isUnverified,
   } = resultDataItem;
   const itemAttributes = new AttributesTemplate();
-  itemAttributes.interpretation = interpretation;
+  itemAttributes.interpretation = interpretationsWithIcons.includes(interpretation);
```

Comment:
Is `itemAttributes.interpretation` suppose to be a true or false value?

Reply:
It ends up becoming used as boolean later in `createStandardResultDisplay` and `createBloodPressureResultDisplay`. Other then that, I didn't find another place where it being a boolean adversely affected what we were using it for.

Reply:
yes - this item attributes object was the start of doing an initialization to save off a series of bools to then use later for quicker yes/no checks.

Could rename object and keys to something like 'hasAttributes.interpretationWithIcon" to be better? Something like:

```suggestion  
const hasAttributes = new AttributesTemplate();
hasAttributes.interpretationWithIcon = interpretationsWithIcons.includes(interpretation);
hasAttributes.comment = hasComment;
hasAttributes.modified = isModified;
hasAttributes.unverified = isUnverified;
```

Reply:
Is it possible to update this key to `hasInterpretationIcon` to indicate it's a boolean value? Is there any downstream impact making this change or is is only used in this component?

Reply:
rather than the just the one key to have `has`, since all the key:values are boolean, rename to something like this?

``` 
const hasAttributes = new AttributesTemplate();
hasAttributes.interpretationIcon = interpretationsWithIcons.includes(interpretation);
hasAttributes.comment = hasComment;
hasAttributes.modified = isModified;
hasAttributes.unverified = isUnverified;
```